src/replication.c

```diff
@@ -853,6 +951,7 @@ int startBgsaveForReplication(int mincapa, int req) {
             /* Keep the page cache since it'll get used soon */
             retval = rdbSaveBackground(req, server.rdb_filename, rsiptr, RDBFLAGS_REPLICATION | RDBFLAGS_KEEP_CACHE);
         }
+        if (server.debug_sleep_after_fork) usleep(server.debug_sleep_after_fork);
```

Comment:
`debug_sleep_after_fork` is measured in seconds, right?
```suggestion
        if (server.debug_sleep_after_fork) sleep(server.debug_sleep_after_fork);
```

Reply:
We can change it to seconds, we anyway use in tests values with 10**6 factor

Reply:
ms sounds better to me

Reply:
👍